### PR TITLE
fix: compliance requirements information

### PR DESCRIPTION
**Pull request by lucasichen**
# GitHub Ticket / Description

## Type && Checklist

- [ ] **New feature (new APIs, new functional methods)**
  - [ ] Created (or updated) unit test cases (if required)
  - [ ] Passed unit testing for **ALL** cases
  - [ ] Created (or updated) E2E cases (if required)
  - [ ] Passed E2E testing for **ALL** cases (local or/and sandbox)
- [x] **Refactoring / Enhancement**
  - [ ] This is breaking change
  - [ ] Created (or updated) unit test cases (if required)
  - [ ] Passed unit testing for **ALL** cases
  - [ ] Created (or updated) E2E cases (if required)
  - [ ] Passed E2E testing for **ALL** cases (local or/and sandbox)
- [ ] **Bug-fix**
  - [ ] Type:
    - [ ] Code Issue:
    - [ ] Business Logic Issue:
  - [ ] Created (or updated) unit test cases (if required)
  - [ ] Passed unit testing for **ALL** cases
  - [ ] Created (or updated) E2E cases (if required)
  - [ ] Passed E2E testing for **ALL** cases (local or/and sandbox)

## Dependency

[Are there any required updates which cannot be done in this repo, or must be done after completing this PR?]

## Choose proper operation and replace the \<placeholder\>: `+semver:<placeholder>`

`breaking` `major` `feature` `minor` `fix` `patch`

+semver:fix

### CODE REVIEW

File: docs/SRS-Volere/SRS.tex [hunk 1]
```diff
@@ -466,37 +466,35 @@ \subsection{Legal Requirements}
...
-    \begin{itemize}
-        \item \textbf{AODA Compliance}: Designing the user interface in accordance
-        with the \textit{Accessibility for Ontarians with Disabilities Act} (AODA) and the 
-        \textit{Integrated Accessibility Standards Regulation} (IASR) to ensure it is
-        perceivable, operable, understandable, and robust for all users.
-        \item \textbf{WCAG 2.1 Compliance}: Ensuring that the tool meets the Web Content 
-        Accessibility Guidelines (WCAG) 2.1 Level AA standards, as required under AODA.
-    \end{itemize}
+    \item \textbf{Software Development Standards}: While full adherence to comprehensive standards 
+    such as ISO/IEC 25010 may be beyond the scope of this student capstone project, we aim to follow
```
Comment (lilweege) [inline docs/SRS-Volere/SRS.tex:470]: Why mention this if we aren't doing it?

File: docs/SRS-Volere/SRS.tex [hunk 2]
```diff
@@ -466,37 +466,35 @@ \subsection{Legal Requirements}
...
+            \item \textbf{OWASP Guidelines}: Basic security measures will be implemented in line with the 
+            Open Web Application Security Project (OWASP) guidelines to mitigate common vulnerabilities such 
+            as injection attacks and unauthorized access.
+            \item \textbf{Secure Database Storage}: All data will be securely stored in a trusted cloud 
+            database service (e.g., AWS), utilizing built-in security features such as encryption at rest 
+            and in transit, to protect sensitive user information.
+        \end{itemize}
+
+
+    \item \textbf{Accessibility Standards}: While the tool may not be fully accessible for users with disabilities,
```
Comment (lilweege) [inline docs/SRS-Volere/SRS.tex:494]: Why not?

File: docs/SRS-Volere/SRS.tex (hunk 1 above)
Comment (lucasichen) [reply docs/SRS-Volere/SRS.tex]: ill saw SOLID principles instead

File: docs/SRS-Volere/SRS.tex (hunk 2 above)
Comment (lucasichen) [reply docs/SRS-Volere/SRS.tex]: added basic accessibility

**Review by lilweege — APPROVED** [review]

**Pull request merged**